streaming/src/protobuf/streaming.proto

```diff
@@ -2,6 +2,8 @@ syntax = "proto3";
 
 package ray.streaming.proto;
 
+import "google/protobuf/any.proto";
```

Comment:
`any` is less structural, can we avoid using it?

Reply:
I'll try to remove this in separate PR.